components/style/binding_tools/regen.sh

```diff
@@ -34,4 +34,4 @@ else
   LIBCLANG_PATH="$(brew --prefix llvm38)/lib/llvm-3.8/lib"
 fi
 
-"./${TOOLS_DIR}/regen.py" --target all "${@}"
+"./${TOOLS_DIR}/regen.py" --target bindings "${@}"
```

Comment:
Oh, that was a mistake. I do this locally since regen.py doesn't set up the env vars.